src/test/unit_tests/nc_coretest.js

```diff
@@ -0,0 +1,450 @@
+        uid: options.nsfs_account_config.uid,
+        gid: options.nsfs_account_config.gid,
+        access_key: options.access_key,
+        secret_key: options.secret_key
+    };
+    const res = await exec_manage_cli(nc_nsfs_manage_entity_types.ACCOUNT, nc_nsfs_manage_actions.ADD, cli_options);
+    const json_account = JSON.parse(res);
+    const account = json_account.response.reply;
+    account.access_keys[0] = {
+        access_key: new SensitiveString(account.access_keys[0].access_key),
+        secret_key: new SensitiveString(account.access_keys[0].secret_key)
+    };
+    return account;
+}
+
+/**
+ * read_account_manage reads an account using manage_nsfs and returns rpc-like result
+ * @param {object} options
+ * @return {Promise<object>}
+ */
+async function read_account_manage(options) {
+    const cli_options = { name: get_name_by_email(options.email), show_secrets: true };
+    const res = await exec_manage_cli(nc_nsfs_manage_entity_types.ACCOUNT, nc_nsfs_manage_actions.STATUS, cli_options);
+    const json_account = JSON.parse(res);
+    const account = json_account.response.reply;
+    account.access_keys[0] = {
+        access_key: new SensitiveString(account.access_keys[0].access_key),
+        secret_key: new SensitiveString(account.access_keys[0].secret_key)
+    };
+    return account;
+}
+
+/**
+ * delete_account_manage deletes an account using manage_nsfs
+ * @param {object} options
+ * @returns {Promise<void>}
+ */
+async function delete_account_manage(options) {
+    const cli_options = { name: get_name_by_email(options.email) };
+    await exec_manage_cli(nc_nsfs_manage_entity_types.ACCOUNT, nc_nsfs_manage_actions.DELETE, cli_options);
+}
+
+/**
+ * delete_account_by_property_manage lists accounts by property and deletes the result list using manage_nsfs
+ * @param {object} params
+ * @param {object} params.nsfs_account_config
+ * @returns {Promise<void>}
+ */
+async function delete_account_by_property_manage({ nsfs_account_config }) {
+    const list = await exec_manage_cli(nc_nsfs_manage_entity_types.ACCOUNT, nc_nsfs_manage_actions.LIST, nsfs_account_config);
+    const accounts = JSON.parse(list).response.reply;
+    for (const account of accounts) {
+        const cli_options = { name: account.name };
+        await exec_manage_cli(nc_nsfs_manage_entity_types.ACCOUNT, nc_nsfs_manage_actions.DELETE, cli_options);
+    }
+}
+
+/**
+ * list_accounts_manage lists accounts using manage_nsfs and returns rpc-like result
+ * @param {object} options
+ * @return {Promise<object>}
+ */
+async function list_accounts_manage(options) {
+    const list = await exec_manage_cli(nc_nsfs_manage_entity_types.ACCOUNT, nc_nsfs_manage_actions.LIST, options);
+    return { accounts: JSON.parse(list).response.reply };
+}
+
+/**
+ * update_account_s3_access_manage updates an account using manage_nsfs
+ * @param {object} options
+ * @returns {Promise<void>}
+ */
+async function update_account_s3_access_manage(options) {
```

Comment:
@romayalon are we validating this account update anywhere else? just a doubt

Reply:
on the tests, this function equal to the rpc function - update_account_s3_access(), the difference here is that on Non containerized version we don't have RPC server so we do the update by running manage NSFS cli calls instead and return rpc like response.